core/components/atoms/radio/Radio.tsx

```diff
@@ -41,6 +41,10 @@ export interface RadioProps extends BaseProps, OmitNativeProps<HTMLInputElement,
    * Denotes selection in case of Controlled `Radio`
    */
   checked?: boolean;
+  /**
+   * Denotes error in case of errors
```

Comment:
Please change this description with `Shows error state in case of failed validation`